packages/libs/components/src/plots/Barplot.tsx

```diff
@@ -121,7 +121,12 @@ const Barplot = makePlotlyPlotComponent(
               orientation: orientation === 'vertical' ? 'v' : 'h',
               opacity: calculatedOpacity,
               type: 'bar',
-              text: showValues ? el.value : undefined,
+              // two decimal points for y values
+              text: showValues
+                ? el.value.map(
+                    (value: number) => `${Number.parseFloat(value.toFixed(2))}`
```

Comment:
I think some unnecessary type conversions can be removed.

```suggestion
                    (value: number) => value.toFixed(2)
```

Reply:
Hi @dmfalke The reason why I introduced Number.parseFloat() is that I wanted to remove unnecessary zeros added to an integer value. For example, for the value of 10, 10.toFixed(2) becomes 10.00. I think it is better to show 10 only in that case?

Reply:
Ah, that makes sense. Thanks for the explanation.

Reply:
My pleasure 😃 Thank you for your review again 👍